cli/orchestration/outputs-sharing.md

```diff
@@ -0,0 +1,182 @@
+---
+title: Configure stack inputs and outputs.
+description: Terramate supports connecting stacks inputs and outputs.
+---
+
+# Introduction
+
+_Outputs Sharing_  is an advanced feature that uses code generation and 
```

Comment:
Is it necessary to use the phrase 'advanced feature' ?

Reply:
it's not that easy to understand and it's not the usual recommended way of solving the problem, so users must be aware (somehow) that choosing this method requires an advanced knowledge of Terramate and Terraform. Feel free to propose a better wording.

Reply:
That makes sense, it indicates this point well.